src/System Application/App/Extension Management/src/ExtensionManagement.Page.al

```diff
@@ -208,37 +215,15 @@ page 2500 "Extension Management"
                         CurrPage.Update(false);
                     end;
```

Comment:
removing the action is a breaking change. You would have to obsolete it, add preprocessor symbols and hide it + make it do nothing...

Reply:
thank you